eventfiles/models.py

```diff
@@ -0,0 +1,29 @@
+import os
```

Comment:
maybe you can import the path like:
```suggestion
from os import path
```